src/SourceBuild/content/repo-projects/Directory.Build.props

```diff
@@ -16,6 +16,7 @@
 
     <ProjectDirectory>$([MSBuild]::NormalizeDirectory('$(SrcDir)', '$(RepositoryName)'))</ProjectDirectory>
     <PackagesOutput>$([MSBuild]::NormalizeDirectory('$(ProjectDirectory)', 'artifacts', 'packages', '$(Configuration)', 'NonShipping'))</PackagesOutput>
+    <RepoManifestFile>$(ProjectDirectory)artifacts/RepoManifest.xml</RepoManifestFile>
```

Comment:
nit: move this into the repo-projects/Directory.Build.targets file at the top and make is use the `NormalizePath` msbuild function.

Reply:
Fixed with https://github.com/dotnet/installer/pull/18153/commits/9bfacadeccfd5235e7e99c2d624cb92b7807b176